pkg/metrics/policystatusmetrics/policystatusmetrics.go

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package policystatusmetrics
```

Comment:
Is there a benefit in having the metrics code in a separate package rather than `pkg/sensors`? This is how other metrics are defined, but I don't love this pattern as it's not immediately obvious what code is instrumented.

Reply:
I cannot really answer for that one, I was not familiar with this and I just followed the pattern.

Reply:
The main issue would be import cycles if the code is moved under `pkg/sensors` - but I don't think it's the case here?

Reply:
I'm not sure I didn't have those issues writing this but cannot really say sorry :(